src/main/java/nl/tudelft/contextproject/script/UpdateTask.java

```diff
@@ -0,0 +1,20 @@
+package nl.tudelft.contextproject.script;
+
+import nl.tudelft.contextproject.ContextTFP;
+
+import java.util.TimerTask;
+
+public class UpdateTask extends TimerTask {
```

Comment:
I agree with @bvobart, adding a seperate class for this should not be necessary and reduces the readability of the code